gateway/up.go

```diff
@@ -18,6 +19,13 @@ func constructReq(ctx context.Context, req *entity.UpRequest) (*http.Request, er
 		return nil, err
 	}
 	httpReq.Header.Set("Content-Type", "multipart/form-data")
+
+	version := constants.Version
+	if constants.IsDevVersion() {
+		version = "dev"
```

Comment:
Do we still want to send the version in dev so that we can test things on Backboard that might be using it? Could add an additional header for variant if we need to check dev/staging/prod

Reply:
As far as I know there is no way to get the version in dev. The version is injected at compile time in CI based off the git tag.

Reply:
Oh right, good point. That's too bad but understandable